src/containers/Header/NotificationsTray/style.postcss

```diff
@@ -0,0 +1,123 @@
+@import 'main.postcss';
+
+.NotificationsTray {
+  margin-right: var(--spaceBase);
+
+  &_icon svg {
+    height: 18px;
+    width: 18px;
```

Comment:
If you set the `font-size`, the inner `width` and `height` will be automatically set (they are both set to `1em`).